src/06-loops.js

```diff
@@ -165,7 +226,20 @@ function timesIndex(array) {}
  *  //> [ 5, 5 + 2, 5 + 2 + 9 ]
  *  //> [ 5, 7, 16 ]
  */
-function cumulativeSum(array) {}
+function cumulativeSum(array) {
+  let newArray = [array[0]];
+  //let emptyArray = JSON.parse(JSON.stringify(array));
```

Comment:
Remember to remove commented out code before pushing.  This is another good habit to solidify as we take on more and more complex projects.  Also - nice work with `JSON.parse()` and `JSON.stringify()`.  This a great example of making a 'deep copy' of an object.  This is definitely something worth taking a little time to really understand what is going on under the hood.